server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTest.java

```diff
@@ -77,7 +77,7 @@
     createSecret(oldName);
 
     String newName = UUID.randomUUID().toString();
-    assertEquals(200, renameSecret(oldName, newName));
```

Comment:
Should we also test that the URL is as expected?

Reply:
Yeah, we probably should, I'll get to that shortly.